aw_server/api.py

```diff
@@ -9,6 +9,7 @@
 import iso8601
 import urllib.request
 import base64
+import requests
```

Comment:
Do we have that lib installed in this app? I remember doing it with urllib because requests wasn't installed.

Reply:
Yes.requests is already installed in this app.